python/Ganga/new_tests/GPI/Tasks/TestCoreTasks.py

```diff
@@ -85,3 +95,30 @@ def test_d_TaskStillRunning(self):
         assert t.status == "running"
         assert t.transforms[0].status == "running"
 
+    def test_e_CreateManyTasks(self):
+        """Create a load of tasks"""
+        from Ganga.GPI import tasks
+
+        for _ in range(0, self._numTasks):
+            t = self.createTask()
+            t.run()
+            
+        assert len(tasks) == self._numTasks + 1
+
+    def test_f_CheckAccessToManyTasks(self):
+        """Access the Tasks while monitoring"""
+        from Ganga.GPI import tasks, CoreTask, CoreTransform, Executable, GenericSplitter
+        from Ganga.GPIDev.Base.Proxy import isType
+
+        for _ in range(0, self._numTasks):
+            for t in tasks:
```

Comment:
Just to keep hammering the access to Tasks while the monitoring is loading. I guess I can just replace this with a random number but I wanted it to scale with the number of Tasks that are being generated.

As I say though, I can't get this to fail locally but should fail make the failure more obvious on the testing machines.